### ISSUE TITLE
New modules need a way to get maintainer info into maintainers file.

### ISSUE THREAD

**Issue by robynbergeron** [ansible/ansibullbot#19]
Pretty sure this is not happening automagically just yet.

**Comment by gregdek** [comment]
It's not.

The clever way to do this would be to look in a module, see the "author", and then plug it in -- but I don't think people populate the author field consistently enough for this to work. Still, if we can do it, we should.

**Comment by gundalow** [comment]
We could maybe enforce the setting of the the author with Sivel's `ansible-validate-module`?

This would just need two initial PRs (one for modules-core, one for modules-extras) to ensure existing modules are valid.

I believe the correct form should be:

`author: "First Last (@github)"`

Though some people seem to have email address.

**Comment by gregdek** [comment]
I think this is a setting that has outlived its usefulness, tbh.

The only reason we added "author" as a necessary field was to help with triage, and we've obsoleted that with the maintainers files.

We could theoretically use this for populating the initial values in the maintainer file, which would make sense. But that process is semi-manual now anyway, and I frequently just use git blame to figure out the initial maintainer anyway.
